assets/src/stories-editor/helpers/index.js

```diff
@@ -2132,3 +2132,14 @@ export const startAnimation = ( block, animationType, animationDuration, animati
 
 	blockElement.addEventListener( 'animationend', callback, { once: true } );
 };
+
+/**
+ * Check if block is page block.
+ *
+ * @param {string} clientId Block client ID.
+ * @return {boolean} Boolean if block is / is not a page block.
+ */
+export const isStoryBlock = ( clientId ) => {
```

Comment:
"Check if block is page block."

Then this should better be named `isPageBlock`.

Reply:
Fixed https://github.com/ampproject/amp-wp/pull/3246/commits/ce6b984e34ce14bbd74859fd9c037066cab8799f